posts/2023-05-24-api-automation-testing-tools-comparison.adoc

```diff
@@ -0,0 +1,320 @@
+= API Automation Testing Tools, a comparison
+TehreemFatima07
+v1.0, 2023-05-24
+:title: API Automation Testing Tools, a comparison
+:imagesdir: ../media/2023-05-24-api-automation-testing-tools-comparison
+:lang: en
+:tags: [api, test automation, testing techniques]
+
+
+Internal projects at Lunatech play a crucial role in fostering growth, innovation, and knowledge-sharing within employees. These projects serve as the sandbox where new joiners are given the opportunity to explore, learn, and contribute in a hands-on manner.
+++++
+</br>
+++++
+Lunagraph is one of the internal projects that will revolutionize how we handle three crucial elements: People, Clients, and Projects. This innovative solution serves as the "source of truth" for these key elements, providing a reliable and consolidated platform that offers immense value to our internal clients.
+++++
+</br>
+++++
+Lunagraph's vision is to be able to reliably service internal clients with a single API integration, previously many apps were used for this integration. One of the task for this project is to have automated testing of APIs for faster test maintenance and refactoring.
+
+=== What is an API Testing?
+API testing refers to the practice of testing Application Programming Interfaces (APIs) to ensure their functionality, reliability, security, and performance. It allows software teams to improve APIs without the worry of introducing bugs or disrupting current functionalities. The process of upgrading code dependencies becomes easier when API verification can be performed automatically.
+
+To start with the API Testing, first step was to select the API Testing tool. This blog post explores the API Testing tools, to present the strengths and weakness of them and to give the comparison that which tools best fits the purpose of API Testing of Lunagraph
```

Comment:
Is the blog post about API testing tools, or API automation testing tools? As I feel those can have different meanings, especially since the tools described mostly programming frameworks, so they are more leaning to "automation" part, no?

Reply:
Its about "API Automation Testing" a relevant line for this part has also been added